graph/loaders/comments.js

```diff
@@ -218,6 +218,14 @@ const getCommentCountByQuery = (context, {ids, statuses, asset_id, parent_id, au
     query = query.where({author_id});
   }
 
+  if (tag) {
+    query = query.find({
+      'tags.tag.name': {
+        $in: tag,
```

Comment:
I think that `tag` should be `tags`. With regards to the performance issues, I think we may need to approach that later in a performance related PR so we can define our indexes.